nautobot/extras/templates/extras/job_list.html

```diff
@@ -7,61 +7,74 @@
             $(this).children("i").toggleClass("mdi-chevron-right mdi-chevron-down");
         });
     </script>
+    <script>
+	$(".accordion-toggle").click(function() {
+	    $(this).toggleClass("mdi-chevron-down mdi-chevron-right");
+        });
+    </script>
 {% endblock %}
 
 {% block content %}
     <h1>{% block title %}Jobs{% endblock %}</h1>
     <div class="row">
         <div class="col-md-12">
             {% if jobs %}
-                <table class="table table-hover table-headings reports">
-                    <tr>
-                        <th>Module / Job</th>
-                        <th>Description</th>
-                        <th class="text-right">Last Run</th>
-                        <th style="width: 40px"></th>
-                        <th style="width: 100px">Last Status</th>
-                        <th style="width: 180px">Last Results</th>
-                    </tr>
+	        <div class="panel-group" id="accordion">
+		    <div class="row panel">
+                        <div class="col-md-2">Module / Job</div>
+                        <div class="col-md-4">Description</div>
+                        <div class="col-md-2 text-right">Last Run</div>
+                        <div class="col-md-2">&nbsp;</div>
+                        <div class="col-md-1">Last Status</div>
+                        <div class="col-md-1">Last Results</div>
```

Comment:
using `col-md-*` here means that these will convert to full-page-width columns in sufficiently narrow browser windows - that seems undesirable to me? I'll try and find some time tomorrow to experiment with this.